Migration.md

```diff
@@ -0,0 +1,285 @@
+# Migrate from Lob’s JavaScript to TypeScript SDK
+
+This guide illustrates differences between Lob’s current JavaScript library (lob-node) and our new TypeScript SDK (lob-typescript-sdk). The new TypeScript SDK supports all generally available endpoints and has the benefit of being built with TypeScript.
+We have also compared performance and found the new TypeScript SDK is equivalent or improved in most cases with some operations experiencing a consistent 2X improvement.
+In this guide we compare the following method pattern.
+
+- CREATE
+- LIST
+- GET
+- DELETE
+- BULK VERIFY (ADDRESS VERIFICATION)
+- VERIFY (BANK ACCOUNTS)
+## INSTALL
+Similar to lob-node, the lob-typescript-sdk package is available through NPM:
+```
+$ npm install lob
+```
+## IMPORT AND INITIALIZE
+Lob-node uses JavaScript’s built-in require. You initialize by passing your API key as an argument:
+```javascript
+const Lob = require("lob")("YOUR API KEY");
+```
+The new TypeScript SDK uses an ES module import statement and configuration variable for the object containing your API key. This variable is used to instantiate a new configuration:
+```typescript
+import { Configuration } from "lob-typescript-sdk";
+
+const config: Configuration = new Configuration({
+  username: "<<YOUR API KEY HERE>>"
+});
+```
+_Note:_ If you are using the new TypeScript SDK using JavaScript here is the code that you would use for this:
+```javascript
+const { Configuration } = require("@lob/lob-typescript-sdk");
+
+const config = new Configuration({
+  username: process.env.LOB_API_KEY
+});
+```
+This approach uses ES6 destructuring to extract the essential classes from the results of requiring the TypeScript SDK..
+## Error Handling
+In the new Typescript SDK, errors returned by the API are thrown, not returned to an error first callback (See below examples in Compare Create methods). As a result, consumer code does not need to do the work of detecting an error, and can
```

Comment:
The Error Handling section is missing an ending?  it says "detecting an error, and can". Sentence incompelte

Reply:
All revisions made.